code/modules/power/smes.dm

```diff
@@ -1,33 +1,37 @@
-// the SMES
-// stores power
-
 #define SMESMAXCHARGELEVEL 200000
 #define SMESMAXOUTPUT 200000
 #define SMESRATE 0.05			// rate of internal charge to external power
 
-
-
 /obj/machinery/power/smes
 	name = "power storage unit"
 	desc = "A high-capacity superconducting magnetic energy storage (SMES) unit."
 	icon_state = "smes"
 	density = TRUE
 
-	var/capacity = 5e6 // maximum charge
-	var/charge = 0 // actual charge
-
-	var/input_attempt = TRUE 		// 1 = attempting to charge, 0 = not attempting to charge
-	var/inputting = TRUE 			// 1 = actually inputting, 0 = not inputting
-	var/input_level = 50000 		// amount of power the SMES attempts to charge by
-	var/input_level_max = 200000 	// cap on input_level
-	var/input_available = 0 		// amount of charge available from input last tick
-
-	var/output_attempt = TRUE 		// 1 = attempting to output, 0 = not attempting to output
-	var/outputting = TRUE			// 1 = actually outputting, 0 = not outputting
-	var/output_level = 50000		// amount of power the SMES attempts to output
-	var/output_level_max = 200000	// cap on output_level
-	var/output_used = 0				// amount of power actually outputted. may be less than output_level if the powernet returns excess power
-
+	/// Maximum charge of the SMES
+	var/capacity = 5e6
+	/// Current charge level
+	var/charge = 0
+	/// Set TRUE if SMES attempting to charge, FALSE if not
+	var/input_attempt = TRUE
+	/// Set TRUE if SMES is inputting, FALSE if not
+	var/inputting = TRUE
+	/// Amount of power the SMES attempts to charge by
+	var/input_level = 50000
+	/// Maximum input level
+	var/input_level_max = 200000
+	/// Charge amount available from input last tick
+	var/input_available = 0
+	// TRUE = attempting to output, FALSE = not attempting to output
+	var/output_attempt = TRUE
+	/// TRUE = actually outputting, FALSE = not outputting
+	var/outputting = TRUE
+	/// Amount of power the SMES attempts to output
+	var/output_level = 50000
+	/// Cap on output_level
+	var/output_level_max = 200000
+	/// Amount of power actually outputted. may be less than output_level if the powernet returns excess power
+	var/output_used = 0
 	var/name_tag = null
 	var/obj/machinery/power/terminal/terminal = null
```

Comment:
Can you autodoc these anyways? And remove the `= null`

Reply:
Oh I missed those. Yeah sure I will squash those real quick.